Integrations/AttackIQFireDrill/AttackIQFireDrill.py

```diff
@@ -316,6 +317,30 @@ def build_test_results_hr(test_results, test_id):
     return tableToMarkdown(f'Test Results for {test_id}', test_results_mod, keys)
 
 
+def get_page_number_and_page_size(args):
+    """
+    Get arguments page_number and page_size from args
+    Args:
+        args (dict): Argument dictionary, with possible page_number and page_size keys
+
+    Returns (int, int): Return a tuple of (page_number, page_size)
+
+    """
+    page = args.get('page_number', 1)
+    try:
+        page = int(page)
+    except (ValueError, TypeError):
+        return_error(f'Error: Invalid page_number value. {page} Is not valid. Please enter a positive integer.')
+    try:
+        page_size = int(args.get('page_size', DEFAULT_PAGE_SIZE))
+    except (ValueError, TypeError):
```

Comment:
return a similar error here if it gets a page size which is not a number or not natural.